### DISCUSSION TITLE
Re-Rank osu!mania Hard Rock mod

### DISCUSSION THREAD

**Discussion by Axieee** [ppy/osu#14971] [Ideas] [answered]
Hi,
So from memory of my younger past and from other older osu players (early 2013-ish), I remember osumania having Hard Rock as a ranked mod, but then it was allegedly unranked because the dev behind it wanted to rework the mod for mania, which one of those changes happened to be potentially adding mines to mania. But the dev left the team shortly after doing this, and as a result the mod has been left under rubbles for many years into the future - some even questioning it's existence. Can this mod finally see the light of day in ranked play? 
Also to note there are several old mania accounts with ranked HR plays, here's an example: https://osu.ppy.sh/users/378480/mania https://osu.ppy.sh/users/2113126/mania

**Comment by peppy** [comment, marked as answer]
![20211006 110445 (dotnet)](https://user-images.githubusercontent.com/191335/136129241-a20dd756-ac74-4fd1-8190-5f3474ace9db.png)

Am I missing something?

**Reply by Axieee** [reply]
I may have messed up one of the options here (as I have no idea how github works), but on the stable release of osu it's unranked whereas lazer might be completely different!

**Reply by mk56-spn** [reply]
it is. afaik ( atleast on standard ) every mod is ranked and submits.

**Reply by bdach** [reply]
this repository is for lazer issues/discussions only.

**Reply by Axieee** [reply]
Yeah I figured I did something wrong here; my bad. I got directed by two other people to post it here so I did

**Reply by smoogipoo** [reply]
The screenshot above doesn't really show anything, but the HR mod is indeed rankable in lazer.
